code/game/objects/items/devices/traitordevices.dm

```diff
@@ -162,17 +162,19 @@ effective or pretty fucking useless.
 	return
 
 /obj/item/device/shadowcloak
-	name = "cloaker belt"
-	desc = "Makes you invisible for short periods of time. Recharges in darkness."
+	name = "Belt mounted Cloaking Device"
+	desc = "An utility belt. There are lots of wires attached to it and theres some sort of tiny \
```

Comment:
A ton of errors in the desc and not really what you'd put in a desc. Make it describe what it does, not what it looks like.